e2e-tests/README.md

```diff
@@ -9,21 +15,31 @@ curl -s https://get.helm.sh/helm-v3.2.4-linux-amd64.tar.gz \
     | tar -C /usr/bin --strip-components 1 -zxvpf - '*/helm'
 curl https://sdk.cloud.google.com | bash
 ```
-MacOS
+
+### MacOS
+
 ```
 brew install coreutils gnu-sed jq kubernetes-cli openshift-cli kubernetes-helm percona-xtrabackup yq
 curl https://sdk.cloud.google.com | bash
 ```
-## With DockerHub
-### Build image
+
+## Building and testing the Operator with DockerHub
+
+Use the following script to build the image:
+
```

Comment:
contributors do not have rights to push docker images to our 'perconalab/percona-xtradb-cluster-operator' organisation/repository. We need to describe here how to use some other account/repository  to  push built docker image:
If `IMAGE` env variable is declared, it will be used as an operator image for build needs and also tests will use this image.  
`export IMAGE=bob/my_repository_for_test_images :K8SPXC-622-fix-feature-X`

Also, user can re-declare all default images which are used for testing. The full list of vars is:
```
IMAGE_PXC=${IMAGE_PXC:-"perconalab/percona-xtradb-cluster-operator:main-pxc8.0"}
IMAGE_PMM=${IMAGE_PMM:-"perconalab/pmm-client:dev-latest"}
IMAGE_PROXY=${IMAGE_PROXY:-"perconalab/percona-xtradb-cluster-operator:main-proxysql"}
IMAGE_HAPROXY=${IMAGE_HAPROXY:-"perconalab/percona-xtradb-cluster-operator:main-haproxy"}
IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-xtradb-cluster-operator:main-pxc8.0-backup"}
IMAGE_LOGCOLLECTOR=${IMAGE_LOGCOLLECTOR:-"perconalab/percona-xtradb-cluster-operator:main-logcollector"}
```

Also, we have the possibility of running operator for the tests in a cluster wide mode. For it the following var should be declared:
`export OPERATOR_NS='pxc-operator`

Also, the following var can be useful (delete all namespaces automatically except   "^kube-|^default|Terminating|pxc-operator|openshift").  
`export  CLEAN_NAMESPACE=1`
But please NOTE all namespaces will be deleted. It is useful in case you run a separate cluster only for the tests.